Backend/src/routes/api.js

```diff
@@ -1,35 +1,21 @@
 import { Router } from "express";
 import net from "net";
-import {spawn} from "child_process"; // TODO
+import { spawn } from "child_process";
 import * as fs from "fs";
 const nReadlines = require('n-readlines');
 import DATA_FORMAT from "../../Data/sc1-data-format/format.json";
 import INITIAL_SOLAR_CAR_DATA from "../../Data/dynamic_data.json";
-import SESSION_SOLAR_CAR_DATA from "../../Data/session_data.json";
 import INITIAL_FRONTEND_DATA from "../../Data/cache_data.json";
-// TODO ---------------------------------------------------------------------
-let doRecord = false; // Flag for whether we should be recording data or not
-let currentSession = ""
 
 const ROUTER = Router();
-let solarCarData = INITIAL_SOLAR_CAR_DATA
-let sessionSolarCarData = SESSION_SOLAR_CAR_DATA
+let solarCarData = INITIAL_SOLAR_CAR_DATA;
 let frontendData = INITIAL_FRONTEND_DATA;
 
-let RecodedData = null
-let sessionsList = []
-
-const broadbandLines = new nReadlines('./recordedData/sessionsList.bin');
-let line;
-let lineNumber = 1;
-
-// Getting all the created sessions from sessionsList.bin
-while (line = broadbandLines.next()) {
-  // console.log(`Line ${lineNumber} has: ${line.toString('ascii')}`);
-  sessionsList.push(line.toString('utf8').replace('\r', ""))
-  lineNumber++;
+let bytesPerPacket = 0;
+for (const property in DATA_FORMAT) {
+  bytesPerPacket += DATA_FORMAT[property][0];
```

Comment:
Change 0 to be a const labelling why it's the 0th index

Reply:
Addressed